cerberus-api-tests/src/main/groovy/com/nike/cerberus/api/CerberusCompositeApiActions.groovy

```diff
@@ -33,10 +33,23 @@ class CerberusCompositeApiActions {
         // Read that the node was updated
         def resp2 = readSecretNode(path, cerberusAuthToken)
         assertEquals(value2, resp2?.'data'?.'value')
+
+        // Since the versions are sorted by timestamp, sleep is used to ensure correct order
+        sleep(1000)
         // Delete the node
         deleteSecretNode(path, cerberusAuthToken)
         // Verify that the node was deleted
         assertThatSecretNodeDoesNotExist(path, cerberusAuthToken)
+        def resp3 = getSecretNodeVersionsMetadata(path, cerberusAuthToken)
```

Comment:
I think this method is doing too much. I would separate the versioning logic into a separate method and have it run as a separate test.